http://lists.pld-linux.org/mailman/pipermail/pld-devel-en/2018-February/025504.html

$ localedb-gen --locales et_EE.UTF-8/UTF-8 -d .
Generating et_EE.UTF-8 using charset UTF-8... failed to set locale!
[error] LC_COLLATE: missing `reorder-end' keyword
failed to set locale!
[error] no output file produced because errors were issued

source commit:
https://sourceware.org/git/?p=glibc.git;a=commitdiff;h=de9661d6beb74b177cef07fe44c90c4e48a983a2

related bugreport:
https://sourceware.org/bugzilla/show_bug.cgi?id=22517
--- glibc-2.27/localedata/locales/et_EE	2018-02-01 18:17:18.000000000 +0200
+++ glibc-2.27/localedata/locales/et_EE	2018-02-16 13:00:00.000000000 +0200
@@ -103,6 +103,8 @@
 <U00FC> <u-diaresis>;<BAS>;<MIN>;IGNORE % ü
 <U00DC> <u-diaresis>;<BAS>;<CAP>;IGNORE % Ü
 
+reorder-end
+
 END LC_COLLATE
 
 LC_CTYPE
